torch_np/_ufuncs.py

```diff
@@ -2,13 +2,14 @@
 
 import torch
 
-from . import _helpers
-from ._detail import _binary_ufuncs
+from . import _binary_ufuncs_impl, _helpers, _unary_ufuncs_impl
```

Comment:
These `_{unary,binary}_ufuncs_impl.py` shoudl probably be merged, but we can do this in the next PR, where we look at `_helpers`.

Reply:
Merging them is tempting indeed. However the current split has a nice effect of being able for write
`for name in dir(_binary_ufuncs_impl)` without introducing an extra list of names.

Reply:
fair enough